society-of-biblical-literature-fullnote-bibliography.csl

```diff
@@ -313,20 +332,76 @@
     </choose>
   </macro>
   <macro name="collection-title">
-    <text variable="collection-title" form="short"/>
-    <text variable="collection-number" prefix=" "/>
+    <!--TODO: Macro is identical to collection-title-note. Seperate macros not needed?-->
+    <choose>
+      <if variable="volume number-of-volumes" match="any">
+      <!-- Presence of Volume info indicates multivolume work rather than series -->
+          <choose>
+            <!--TODO: Is this needed? Creates double "in".-->
```

Comment:
what's the double "in" referring to? We'd almost certainly not want that, so when does it occur?

Reply:
I forgot to remove this comment once I fixed the error.  I'll remove comments which are no longer relevant via a new commit below.

Reply:
Fixed with commit f3269e6.